_pages/home.md

```diff
@@ -22,30 +22,65 @@ scientific Python packages. We also build technical capacity by providing a
-
-## Who's Involved
-
-pyOpenSci is a new effort being lead by a small team of Python enthusiasts including:
-
-* Leah Wasser, Director of Earth Analytics Education -- Earth Lab, University of Colorado -- Boulder.
-* Max Joseph, Data Scientist -- Earth Lab, University of Colorado -- Boulder.
-* Jenny Palomino, Faculty and Curriculum Developer -- Earth Lab, University of Colorado -- Boulder.
-* Chris Holdgraf, Project Jupyter, Berkeley - BIDS.
-* Luiz Irber, DIB Lab -- UC Davis
-* Paige Bailey, Google
-* Kylen Solvik, Graduate Student -- Earth Lab, University of Colorado -- Boulder.
-* Carson Farmer, External advisor -- Textile.io.
-* Leonardo Uieda, Visiting Researcher -- University of Hawai'i at Mānoa -- Honolulu.
-* Filipe Fernandes, IOOS
+1. **[Join a community meeting](#community-meetings)**: We have monthly video meetings that you are welcome to attend.
+2. **[Join our community forum](https://pyopensci.discourse.group/)**. This is an online space
+   for discussion within the pyOpenSci community. Announcements for new meetings will be posted here.
+3. **Submit a package to pyOpenSci**. We are looking for Python packages that
+   support scientific Python community. Read more about our
+   [review process and scope](https://www.pyopensci.org/dev_guide/peer_review/aims_scope.html)
+4. **Volunteer to be a reviewer for pyOpenSci**. We are looking for people who
+   can help review software packages as they are submitted.
+5. **Help with technical infrastructure**. pyOpenSci has a few pieces of technical
+   infrastructure (like this site) which need maintenance and development.
+6. **Spread the word**! pyOpenSci is a community-driven project. Tell your friends,
+   recommend that a colleague submit a package, mention us at conferences, spread the word!
+
+### Community meetings
+
+Currently, the pyOpenSci team holds community meetings roughly every 2 weeks. These are
+held remotely, and open to anyone who would like to attend. We discuss ideas for improving
+the process / technology / community management / etc of the pyOpenSci community, and
+would value your participation!
+
+The best place to find information about the next community meeting is
+[in the community forum](https://pyopensci.discourse.group). We post dates / times
+for new meetings there.
+
+### Who's Involved
+
+See our [contributors page](contributors.html) for a list of the contributors
+in the pyOpenSci community.
+
+## FAQ
+
+### What's the difference between pyOpenSci and JOSS?
+
+The [Journal of Open Source Software](https://joss.theoj.org/) is a community dedicated
+to improving the visibility and quality of scientific software. They do
+so by providing a lightweight review and publishing process so that authors
+of packages can publish their packages with a DOI and citable artifact.
+They intentionally try to avoid lengthy review and feedback cycles
```

Comment:
I'm not sure this is quite right. I'd rather see this phrased something like:

"JOSS reviews are limited in scope and the [submission criteria](https://joss.readthedocs.io/en/latest/review_criteria.html) less stringent than those of pyOpenSci."